<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChordDiagramDialog</class>
 <widget class="QDialog" name="ChordDiagramDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="windowTitle">
   <string>Chord Diagram</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::FieldsStayAtSizeHint</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="chordNameLabel">
       <property name="text">
        <string>Chord Name:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="ClickableLabel" name="chordNameButton" native="true"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="topFretLabel">
       <property name="text">
        <string>Top Fret:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="topFretSpinBox"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="numStringsLabel">
       <property name="text">
        <string>Number of Strings:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="numStringsSpinBox"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="diagramLabel">
       <property name="text">
        <string>Diagram:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QGraphicsView" name="diagramView">
       <property name="maximumSize">
        <size>
         <width>180</width>
         <height>200</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ClickableLabel</class>
   <extends>QWidget</extends>
   <header>widgets/clickablelabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
